homeassistant/components/nest/camera_sdm.py

```diff
@@ -33,18 +36,20 @@ async def async_setup_sdm_entry(
             CameraImageTrait.NAME in device.traits
             or CameraLiveStreamTrait.NAME in device.traits
         ):
-            entities.append(NestCamera(device))
+            entities.append(NestCamera(hass, device))
     async_add_entities(entities)
 
 
 class NestCamera(Camera):
     """Devices that support cameras."""
 
-    def __init__(self, device: Device):
+    def __init__(self, hass: HomeAssistantType, device: Device):
```

Comment:
Don't pass in `hass`. It will be set on the entity when the entity has been added to home assistant. This means you can access `hass` with `self.hass`, after the entity has been added to home assistant.